### PR TITLE
add chat history flag to onlinemeeting

### PR DESCRIPTION
**Pull request by yuyaolian-msft**

### CODE REVIEW

File: api-reference/beta/resources/onlinemeeting.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
 | participants | [meetingParticipants](meetingparticipants.md) | The participants associated with the online meeting. This includes the organizer and the attendees. |
 | recordAutomatically | Boolean | Indicates whether to record the meeting automatically. |
 | recording | Stream | The content stream of the recording of a [Teams live event](/microsoftteams/teams-live-events/what-are-teams-live-events). Read-only. |
+| shareMeetingChatHistoryDefault | [meetingChatHistoryDefaultMode](#meetingchathistorydefaultmode-values) | Specifies whether meeting chat history is shared with participants. |
```
Comment (Danielabom) [inline api-reference/beta/resources/onlinemeeting.md:59]: Please see step 2 on this wiki and update the enum accordingly: (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations)

File: api-reference/beta/resources/onlinemeeting.md [hunk 2]
```diff
@@ -91,6 +92,14 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
 | limited            | Meeting chat is enabled but only for the duration of the meeting call. |
 | unknownFutureValue | Unknown future value.                                                  |
 
+### meetingChatHistoryDefaultMode values
+
+| Value              | Description                                                            |
+| ------------------ | ---------------------------------------------------------------------- |
+| all                | All meeting chat history is shared.                                    |
+| none               | No meeting chat history is shared.                                      |
+| unknownFutureValue | Unknown future value.                                                  |
```
Comment (Danielabom) [inline api-reference/beta/resources/onlinemeeting.md:101]: Please see step 2 on this wiki and update the enum accordingly: (https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211749/Evolvable-enumerations)

**Review by Danielabom — CHANGES_REQUESTED** [review]
Thank you for working on this PR @yuyaolian-msft. I added a couple of comments, please review them.

**Comment by yuyaolian-msft** [comment]
Hello @Danielabom I have updated the doc per your comment, please let me know if there are any other concerns. Thanks!

**Review by Danielabom — APPROVED** [review]
@yuyaolian-msft , thanks for reviewing my comments. PR approved.

**Comment by Danielabom** [comment]
@yuyaolian-msft, do not forget to add the "ready to merge" label as soon as you are ready to publish.

**Pull request merged**